docs/migration/migrate_7_0.asciidoc

```diff
@@ -119,6 +119,10 @@ Previously set `xpack.monitoring.elasticsearch.ssl.verify` is equal to `xpack.mo
 
 *Impact:* Users with `xpack.monitoring.report_stats` set should use `xpack.xpack_main.telemetry.enabled` instead.
 
+[float]
+=== kibana.yml setting `xpack.monitoring.*.index_pattern` is no longer valid
```

Comment:
Can you pluralize settings in this section or create separate sections for each individual setting. Singular "setting" with the wildcard doesn't read well (to me anyway) but I'll defer to @gchaps or another tech writer for their opinion.

Reply:
<img src="https://media.giphy.com/media/l41lZxzroU33typuU/giphy.gif"/>